.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
`v4` shoudl be preferred (i think v2 will be deprecated soonish, or at least i am seeing warnings in some repos)